src/epub/text/bertie-changes-his-mind.xhtml

```diff
@@ -178,9 +178,9 @@
 			<p>A silvery peal of laughter from the front row stopped him again.</p>
 			<p>“Girls!” said Miss Tomlinson. She spoke in a low, soft voice, but the effect was immediate. Perfect stillness instantly descended upon all present. I am bound to say that, brief as my acquaintance with Miss Tomlinson had been, I could recall few women I had admired more. She had grip.</p>
 			<p>I fancy that Miss Tomlinson had gauged <abbr>Mr.</abbr> Wooster’s oratorical capabilities pretty correctly by this time, and had come to the conclusion that nothing much in the way of a stirring address was to be expected from him.</p>
-			<p>“Perhaps,” she said, “as it is getting late, and he has not very much time to spare, <abbr>Mr.</abbr> Wooster will just give you some little word of advice which may be helpful to you in after-life, and then we will sing the school song and disperse to our evening lessons.”</p>
```

Comment:
These both should also be left as-is because they're not referring to the "afterlife" but rather "after-life"